src/lib/stepper/stepper.scss

```diff
@@ -1,7 +1,118 @@
-.mat-stepper-content[aria-expanded='false'] {
-  display: none;
+$mat-horizontal-stepper-header-height: 72px;
+$mat-vertical-stepper-header-height: 24px;
```

Comment:
are any of these values related to each other or conceptually the same thing? for example I see a lot of `24px`